src/libs/workflow-utils.js

```diff
@@ -54,3 +55,10 @@ export const downloadIO = (io, filename) => {
   const blob = new Blob([JSON.stringify(prepIO(io))], { type: 'application/json' })
   FileSaver.saveAs(blob, `${filename}.json`)
 }
+
+export const downloadWorkflows = (headers, rows, filename) => {
+  const rowsStringified = _.map(row => _.map(v => _.isObject(v) ? JSON.stringify(v) : v, _.values(row)), rows)
```

Comment:
It might be nice to sort the workflow ID as the first column to make looking through the TSV easier.

Reply:
Agreed. Maybe sort by Entity Name as we do in the UI?

Reply:
Added sorting by entity name in cc503c108a4e26fd27b97cc3a794d1e7c1d8b726